Fix isMuted so it doesn't do an extra unit conversion.

picklesrus opened the pull request:
Rename state's muteExpiresAt to muteExpiresAtMs so units are explicit.

picklesrus commented on src/views/preview/comment/compose-comment.jsx:
```diff
@@ -118,7 +118,7 @@ class ComposeComment extends React.Component {
     }
 
     isMuted () {
-        return this.state.muteExpiresAt * 1000 > Date.now();
```
muteExpiresAt in the state object is already being saved in ms in the constructor and in the comment's post response so I'm pretty sure this one is a bug.

chrisgarrity reviewed (commented):
It would be good to figure out a test. It seems as though there are two things we're counting on, redis returns expiration in seconds - that's the value in mute_status. Date wants everything in milliseconds.  Could we set up a test to ensure that `state.muteExpiresAtMs == props.mute_status.muteExpiresAt * 1000`.

It would need a second mockStore that set up the mute_status value.

picklesrus commented:
Good idea. Test added!

chrisgarrity reviewed (approved):
Looks good!

picklesrus commented:
Tests failed after I rebased because of where now mounted (vs shallow mounted) component was looking for the `store`.  Fixed it up in a separate commit. If that passes, I'll merge this.
